README.md

```diff
@@ -19,9 +19,11 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 <a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/readme/images/community.png" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
 
 <p>
-✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">weekly community meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
-✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">Weekly Community Meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
+✔️ <em><strong>Watch</strong></em> Community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Meeting Recordings</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a>.<br />
+✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
+
```

Comment:
@devansh42 : Pl remove this space b/w. `Line 25` & `Line 26`.
Also, signed-off your commit by clicking <a href="https://github.com/layer5io/gowrk2/pull/12/checks?check_run_id=2965655174">here</a>, & simply run the command in your code editor shell.

Reply:
I have made the changes as you asked for.

Reply:
@devansh42 : Also need to signed-off your commit, as said previously :slightly_smiling_face: 
Wanna learn more, go through <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">this</a>.

Reply:
I'm done with signing off the commits. Pardon me for these noob activities.

Reply:
@devansh42: You fixed this, somehow & its LGTM! :partying_face: 
But, I think, you didn't notice that `unverified` signature, Pl check whether you are using right means to commit.

@adithyaakrishna : Sir, pl take this to your end. :slightly_smiling_face:

Reply:
@amino19 I have added my gpg key, it's now verified.